lib/routes/user/get_refund/get_refund_page.dart

```diff
@@ -85,9 +80,9 @@ class _GetRefundState extends State<GetRefund> {
                                 padding: const EdgeInsets.all(16.0),
                                 child: SizedBox(
                                     height: 36.0,
-                                    width: 140.0,
+                                    width: 145.0,
                                     child: SubmitButton(
-                                        "REFUND",
+                                        item.refundBroadcasted ? "BROADCASTED" : "REFUND",
```

Comment:
Maybe SENT is a better choice of word, just a suggestion

Reply:
Technically it is for the state where the transaction is broadcasted and not yet confirmed, hence the name. I will ask Roy for his take here.